### PR TITLE
Copy generation tfms to runtime folder

### PR DESCRIPTION
**Pull request by BrennanConroy**
@pranavkm @NTaylorMullen @DamianEdwards @danroth27 @Eilon

### CODE REVIEW

File: makefile.shade [hunk 1]
```diff
@@ -499,7 +499,9 @@ var CAN_BUILD_ONECORE = '${Directory.Exists(VS_ONECORE_ROOT) && Directory.GetFil
 
     @{
         var dnxCore50Pattern = Path.Combine(BUILD_DIR2, "*", "dnxcore50", "**.*");
+        var dotnet45Pattern = Path.Combine(BUILD_DIR2, "*", "dotnet4.5", "**.*");
```
Comment (NTaylorMullen) [inline makefile.shade]: Nice typing bro

**Comment by NTaylorMullen** [comment]
:watch:

**Comment by BrennanConroy** [comment]
:up: :date:

**Comment by NTaylorMullen** [comment]
:shipit:

**Pull request merged**